app/assets/javascripts/global.js

```diff
@@ -28,6 +28,15 @@ $(document).ready(function() {
     $( ".timepicker" ).datetimepicker({
       format: 'h:mm A'
     });
+
+    //Adds listeners to select inputs to toggle disclaimers on rental schedule page
+    $(document.getElementsByName("rental[item_type_id]")).each(function() {
+      $(this).change(function () {
+        $(this).closest("form").find(".disclaimer").each(function() {
+          $(this).toggleClass("disclaimer-hidden")
+        })
+      })
+    })
```

Comment:
And @Cobchise would you might pushing a merge or a minor commit to the readme? I am thinking travis had a hiccup.